ludwig/models/model.py

```diff
@@ -1416,17 +1416,17 @@ def save_savedmodel(self, save_path):
             inputs[feature['name']] = getattr(self, feature['name'])
 
         for feature in self.hyperparameters['output_features']:
-            outputs[feature['name']] = getattr(
+            outputs['predictions_' + feature['name']] = getattr(
```

Comment:
I changed the output name because of datasets compatibility. Predictions should have different name by design, to  achieve datasets merging (input + output + predictions) smoothly as possible.